android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIImageView.java

```diff
@@ -761,6 +761,18 @@ private void setImageInternal() {
 		}
 	}
 
+	private void cacheImage(TiDrawableReference imageRef) {
+	    synchronized(releasedLock) {
+            int hash = imageRef.hashCode();
+            Bitmap b = mMemoryCache.get(hash);
+            if (b == null) {
+                Log.i(TAG, "Image isn't cached");
+                b = imageRef.getBitmap(true);
+                mMemoryCache.put(hash, b);
```

Comment:
caching an image b/c it flickers the first time it loads doesn't seem like a good solution. Also, we already have a caching system in place...